account-kit/core/src/actions/createAccount.ts

```diff
@@ -266,8 +208,30 @@ function convertAccountParamsToCreationHint<
           salt: toHex(params.accountParams?.salt ?? 0n),
         };
   }
+  if (isLightAccountParams(params)) {
```

Comment:
line 206 here isn't safe btw. `accountParams` can include things that `wallet_requestAccount`'s `creationHint` doesn't support.